packages/sanity/src/core/create/__tests__/createUtils.test.ts

```diff
@@ -69,6 +69,36 @@ describe('createUtils', () => {
       expect(isSanityCreateExcludedType(documentType)).toEqual(false)
     })
   })
+
+  describe('isSanityCreateStartCompatibleDoc', () => {
```

Comment:
what constitutes a sanity create compatible document?

Reply:
(nvm, I meant to delete this before submitting review, found answer to this later on)